src/vaadin-date-time-picker.html

```diff
@@ -78,6 +78,14 @@
   <script>
     (function() {
 
+      /**
+       * @typedef {object} TimePickerTime
+       * @property {string | number} hours
+       * @property {string | number} minutes
+       * @property {string | number} [seconds]
+       * @property {string | number} [milliseconds]
+       */
```

Comment:
This is not needed for TS defs, but I added it as a convenience so that the IDE can help with this type even when working with the Polymer 2 sources since `TimePickerTime` is used internally.